msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/Error.kt

```diff
@@ -79,7 +85,14 @@ open class Error(
     open var exception: Exception? = null,
     open val errorCodes: List<Int>? = null
 ) {
-    fun isBrowserRequired(): Boolean = this.errorType == ErrorTypes.BROWSER_REQUIRED
+}
+
+/**
+ * BrowserRequiredError error is an interface for all errors that could required a browser redirection in Native Auth.
```

Comment:
`required` should be changed to `require`

Reply:
Updated the comment